en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -17,115 +17,114 @@ review-ticket: https://github.com/programminghistorian/ph-submissions/issues/374
+The most direct way to select videos for research is to visit the [YouTube](https://www.youtube.com/) site, and capture a list of video IDs: these are the set of alphanumeric characters that appear in the video's URL, immediately after `watch?v=`. See for example the video ID circled in red in the illustration below: `24xsqyMcpRg`. These IDs are constant and do not change over time. 
 
-For politically salient topics, an ideal dataset will include an equal number of videos from creators representing both the opposing political perspectives, and should have a substantial number of comments (~2000+) to minimize the skew that outlier comments can introduce. Finally, the total comments from each viewpoint should be balanced. In this case, we made sure that comments from the political ‘left’ and ‘right’ were similar in number. 
+{% include figure.html filename="en-or-text-mining-youtube-comments-02.png" alt="Screenshot of YouTube video with video ID in browser link circled in red" caption="Figure 2. Screenshot of YouTube video with video ID in browser link circled in red" %}
 
-## Querying YouTube's API
-We recommend using YouTube Data Tools to query YouTube’s API. 
+For this lesson, we gathered comment data by searching YouTube for 'black lives matter george floyd'. We selected a total of six videos from politically polarized news sources (as ranked by [allsides.com](allsides.com)), from the left-leaning New York Times, Vox, and NBC News, and the right-leaning Daily Mail, Fox News, and the Daily Wire. Choosing multiple videos is often the best approach for the exploratory stages of research, because while YouTube makes available a wide range of metadata about each video (number of likes, title, description, tags and more), the YouTube API may not return comment data for every video searched. For politically salient topics, an ideal dataset will include an equal number of videos from creators representing both the opposing political perspectives, and should have a substantial number of comments (~2000+) to minimize the skew that outlier comments can introduce. Finally, the total comments from each viewpoint should be balanced. In this case, we made sure that comments from the political left and right were similar in number. 
 
-YouTube Data Tools is developed by Bernhard Rieder, Associate Professor in Media Studies at the University of Amsterdam, and supported by the Dutch Platform Digitale Infrastructuur Social Science and Humanities. Rieder maintains and  regularly updates the tool to ensure its continuing compatibility with YouTube’s API.
+>You may prefer to simply [download the sample dataset](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/assets/text-mining-youtube-comments/AllComments__News_v_Comedy_BLM_Coverage.xlsx) and focus on the analysis stage, in which case you can skip the next two sections directly to [Setting Up your Coding Environment](#Set-Up-your-Coding-Environment).

```

Comment:
I added a note to the reader here, letting them know that they can skip the next two sections if they want to use the sample dataset. You had left a similar comment further below, but I thought it would be easier to let them know earlier on.

I just want to double check that the asset referred to is correct, as it seems like you refer to a `.csv` file below, and to `ytdt_data`

Reply:
that sounds good, shouldn't there be a clickable link to download it?

Reply:
The links are click-to-download when the lesson has been published. It is only that things work slightly differently in ph-submissions ☺️